e2e/e2e-helper.js

```diff
@@ -303,6 +303,10 @@ export default class Helper {
   addComponent(filePaths: string = path.normalize('bar/foo.js'), cwd = this.localScopePath) {
     return this.runCmd(`bit add ${filePaths}`, cwd);
   }
+
+  removeFile(filePath: string = path.normalize('bar/foo.js'), cwd = this.localScopePath) {
```

Comment:
There is already a function called deleteFile. 
Remove the remove and use the delete:)